src/react/ui-elements/FormLayout.tsx

```diff
@@ -45,7 +45,7 @@ export const SubTitle = styled.div`
 `;
 export const Fieldset = styled.fieldset`
   display: flex;
-  flex-direction: ${(props) => props.direction || 'column'};
+  flex-direction: ${props => props.flexDirection || 'column'};
```

Comment:
This change is useless, because you are using the props.style cf : 
```js
<F.Fieldset style={{ width: '100%', flexDirection: 'row' }}>
```

If you want really want to do this change you should have something like this : 
```js
<F.Fieldset flexDirection style={{ width: '100%' }}>
```